qdrant-landing/content/documentation/concepts/collections.md

```diff
@@ -578,19 +655,45 @@ automatically be rebuilt in the background to match updated parameters.
 To put vector data on disk for a collection that **does not have** named vectors,
 use `""` as name:
 
+```bash
+curl -X PATCH http://localhost:6333/collections/test_collection1 \
+  -H 'Content-Type: application/json' \
+  --data-raw '{
+    "vectors": {
+        "": { 
+            "on_disk": true 
+      }
+    }
+  }'
+```
+
 ```http
 PATCH /collections/{collection_name}
 {
     "vectors": {
         "": {
             "on_disk": true
         }
-    },
+    }
 }
 ```
 
 To put vector data on disk for a collection that **does have** named vectors:
 
+Note: To create a vector name, follow the procedure from our [Points](/documentation/concepts/points/#create-vector-name).
```

Comment:
I've modified the points.md file in this PR, with the `#create-vector-name` anchor